app/pages/sections/routes.js

```diff
@@ -49,10 +49,7 @@ export const sectionRoutes = (authProvider, addContext, sessionManager) => {
       accessToken,
     });
 
-    if (response.success) {
-      sessionManager.saveToSession({ req, key: 'description', value: req.body.description.trim() });
-      return res.redirect(`${config.baseUrl}/organisation/${response.orderId}`);
-    }
+    if (response.success) return res.redirect(`${config.baseUrl}/organisation/${response.orderId}`);
```

Comment:
Sorry, just thinking about this I think we do need to clear the order description from session here (on success) or update it as you had originally; otherwise, the call to `getOrderDescription` will retrieve the value we might previously have saved to session.

Once that's done, I think covers task 8869 that I was just about to pick up but we may as well do it as part of this change I guess.

Reply:
should I save it as orderDescription or just description?

Reply:
You can use `sessionKeys.orderDescription` from `sessionHelper`.